server/utils/placementApplications/review.test.ts

```diff
@@ -0,0 +1,116 @@
+import type { Request } from 'express'
+import { DeepMocked, createMock } from '@golevelup/ts-jest'
+import type { PlacementApplicationDecision } from '@approved-premises/api'
+import { PlacementApplicationReview } from './review'
+
+describe('PlacementApplicationReview', () => {
+  const id = 'some-uuid'
+
+  afterEach(() => {
+    jest.clearAllMocks()
+  })
+
+  it('initialises with expected values', () => {
+    const request: DeepMocked<Request> = createMock<Request>({
+      params: { id },
+    })
+
+    const review = new PlacementApplicationReview(request, 'review')
+
+    expect(review.applicationId).toEqual(id)
+    expect(review.body).toEqual({})
+    expect(review.step).toEqual('review')
+  })
+
+  describe('validate', () => {
+    const request: DeepMocked<Request> = createMock<Request>({
+      params: { id },
+      session: createMock<Request['session']>({
```

Comment:
I don't think we need to use the `createMock` func here, just an object literal is ok